### PR TITLE
refactor(dialog): dialog close button

### PR DESCRIPTION
**Pull request by inavac182**
## 🔥 UiDialog to use UiIcon for close icon
----------------------------------------------

### Description
Using UiIcon to render the close icon instead of the emoji that was using before.

#### Before
Screenshot 2023-05-15 at 9 24 07 PM

#### After
Screenshot 2023-05-15 at 9 23 55 PM

### CODE REVIEW

File: packages/dialog/docs/docs.mdx [hunk 1]
```diff
@@ -19,14 +19,28 @@ import { DialogsExample } from './utils/';
   </a>
```
Comment (inavac182) [inline packages/dialog/docs/docs.mdx:19]: This is WIP.

File: packages/dialog/src/__private/dialog-toolbar.tsx [hunk 2]
```diff
@@ -29,17 +31,26 @@ const Div = styled.div<UiReactPrivateElementProps>`
...
+  title,
+}: DialogToolbarProps) => {
   const theme = React.useContext(ThemeContext);
 
   return (
     <Div customTheme={theme.theme} selectedTheme={theme.selectedTheme}>
-      {!hideCloseIcon && <Button onClick={closeCB}>❌</Button>}
+      {!hideCloseIcon && (
+        <Button onClick={closeCB} aria-label={closeLabel}>
+          <UiIcon icon="Cross" />
```
Comment (saudelog) [inline packages/dialog/src/__private/dialog-toolbar.tsx:51]: is it worth at some point making the icon configurable ?

**Review by saudelog — DISMISSED** [review]

File: packages/dialog/src/__private/dialog-toolbar.tsx (hunk 2 above)
Comment (inavac182) [reply packages/dialog/src/__private/dialog-toolbar.tsx:51]: That's a good idea, will add it to the backlog.

Comment (inavac182) [reply packages/dialog/src/__private/dialog-toolbar.tsx:51]: https://github.com/inavac182/uireact/issues/156

**Pull request merged**